cmd/minishift/cmd/config/config.go

```diff
@@ -106,6 +106,8 @@ var (
 	CacheImages  = createConfigSetting("cache-images", SetSlice, nil, nil, false, nil)
 
 	// Pre-flight checks (before start)
+	SkipInitialStartupCheck   = createConfigSetting("skip-check-initial-startup", SetBool, nil, nil, true, nil)
+	WarnInitialStartupCheck   = createConfigSetting("warn-check-initial-startup", SetBool, nil, nil, true, false)
```

Comment:
I find the flag names too abstract and hence not intuitive for the user to guess why these config options are present.  Is it just me or someone else also thinks the same cc @minishift/minishift-dev ?

Reply:
These config options sounds similar to https://github.com/minishift/minishift/pull/2325 i.e. about the whole preflight checks.